### DISCUSSION TITLE
Docker-format image in OCI Wrapper?

### DISCUSSION THREAD

**Discussion by andrewm-aero** [containers/buildah#4549] [Q&A] [answered]
I'm not sure if this is a bug, and if so, on whose side, but the [latest dockerhub ubuntu images](https://hub.docker.com/layers/library/ubuntu/focal-20230126/images/sha256-d5d4814ffb155d588f10ec8926d9e1cd09b6d41a3110d2f42959e2fc37f6d0b4?context=explore) (pushed today) are not pull-able by buildah, complaining that they have an `unsupported OCIv1 media type: "application/vnd.docker.container.image.v1+json"`. Docker, on the other hand, pulls it just fine. Taking it apart, it does in fact appear to be an OCIv1 wrapper with an entire docker v1 image as an element, rather than separate layers and configs as you'd expect. 

I've never heard of such a thing. Is this valid, or does this suggest an error on Ubuntu's part? [This code](https://github.com/containers/image/blob/abae97a8306d61cc66addba6b3d0e1ad81d83045/manifest/oci.go#L44) appears to be what determines this (non-)compatibility, and sure enough, that media type is not listed.

**Comment by flouthoc** [comment, marked as answer]
@andrewm-aero This is actually an issue, lets discuss this here https://github.com/containers/buildah/issues/4551

**Reply by flouthoc** [reply]
This is already resolved.
